DrewBradley/rancid-tomatillos
# Feature/react router

### What does this PR do?
Adds Routes to allow conditional rendering, so only the necessary data displays.

### What does it fix?
It removes unnecessary data from MainDisplay and IndividualDisplay.  Uses Switch, Route, and Link to direct the user to the wanted data.

### Is this a feature or a fix?
Feature

### Where should the reviewer start?
App.js, MainDisplay.js, Individual.js, Movie.js, Footer.js.

### How should this be tested?
run ```npm start``` and confirm that the URL changes with the rendering of a new page.
